onelogin_rules/onelogin_unusual_login.yml

```diff
@@ -10,6 +10,10 @@ LogTypes:
 Tags:
   - Identity & Access Management
 Severity: Medium
+SummaryAttributes:
+  - event_type_id
+  - user_id
```

Comment:
Not sure if I missed any of these previously, but it might be good to do `user_name` anytime we do  `user_id` since one is guaranteed to be unique among users but  one is actually human  readable (unless the `user_id` field is human readable as well,  not sure)

Reply:
IDK what `user_id` generally looks like in this context. After reading the docs for OneLogin more closely, I decided to add `account_id` along with the `user_name`. Unclear whether or not `user_id` is human readable as well.
@rleighton

Reply:
From my understanding, `user_id` is a unique int value, `user_name` is human-readable and (potentially) could be reused (e.g. if there were several folks named `Jane Doe`).